packages/forms-web-app/src/dynamic-forms/dynamic-components/add-more/question.js

```diff
@@ -21,14 +21,15 @@ class AddMoreQuestion extends Question {
 	 * @param {Array.<BaseValidator>} [params.validators]
 	 * @param {string} params.viewFolder
 	 */
-	constructor({ title, question, fieldName, hint, validators, viewFolder }) {
+	constructor({ title, question, fieldName, hint, validators, viewFolder, html }) {
 		super({
-			title: title,
-			viewFolder: viewFolder,
-			fieldName: fieldName,
-			question: question,
-			validators: validators,
-			hint: hint
+			title,
```

Comment:
Does this work ok without assigning the values to the parameter names and just doing them in the correct sequence?

Reply:
yep, technically even the order doesn't matter. 
For example:
`constructor({title, question}) {
super({question, title});
}`

Will be interpreted by Javascript as:

`constructor({title, question}) {
super({question: question, title: title});
}`

Reply:
Nice. Thanks Marc, nice bit of knowledge for me to acquire.